pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/signed-exchange.md

```diff
@@ -207,5 +207,7 @@ Within the DevTools console, under the `network` tab, you will be able to see `s
 
 Here is a list of CDNs and hosting providers offering out-of-the-box support for signed exchanges. Using one of these is the easiest way to get started with signed exchanges:
 
+- [AMP Packager on the Google Cloud Marketplace](TODO LINK TO BLOG)
+  [AMP Packager](https://github.com/ampproject/amppackager#amp-packager) is a tool to improve AMP URLs by serving AMP using Signed Exchanges. Follow [our guide to deploy AMP Packager](https://github.com/ampproject/amppackager/tree/releases/deploy/gcloud#amp-packager-google-cloud-deployment) using unix shell scripts and Docker containers.
```

Comment:
We want the link to the Google Cloud Marketplace click-to-deploy installer URL, not the previous way of installing it using shell scripts.  You basically want to mention that the  AMP Packager Google Cloud Click-to-Deploy Installer is now available in the marketplace.

Reply:
gave it an update, lmkwyt!

Reply:
Thanks, the update LGTM!